### PR TITLE
Featuresetup

### PR DESCRIPTION
**Pull request by aliu730**
Part 1 of setup. Database should be written. Working on server/api routes.

### CODE REVIEW

File: database/db.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+			if (err) {
+				console.log("ERR in db.js database", err);
+			} else {
+				//cb the bdy.
+			}
+		});
+	}
+}
+
+// time length is 47
```
Comment (VincentCastelli) [inline database/db.js]: Indentation here is awful. Please fix.

**Review by VincentCastelli — CHANGES_REQUESTED** [review]
Please fix indentation on this file. Thanks!

File: database/db.js [hunk 2]
```diff
@@ -3,26 +3,25 @@ var sample = require('./exampleData.js'); //sample data.
...
 						'10:00 am', '10:30 am', '11:00 am', '11:30 am', '12:00pm',
 						'1:00 pm', '1:30 pm', '2:00 pm',
 						'2:30 pm', '3:00 pm', '3:30 pm', '4:00 pm', '4:30 pm',
 						'5:00 pm', '5:30 pm', '6:00 pm', '6:30 pm', '7:00 pm',
 						'7:30 pm', '8:00 pm', '8:30 pm', '9:00 pm', '9:30 pm',
-						'10:00 pm', '10:30 pm', '11:00 pm', '11:30 pm'];
-var q = 'INSERT INTO bookings (party_size, save_date, table_time, restaurant_id) VALUES (?, ?, ?, ?)';
+						'10:00 pm', '10:30 pm', '11:00 pm'];
+var q = 'INSERT INTO bookings (party_size, date, time, restaurant_id) VALUES (?, ?, ?, ?)';
 for (var i = 0; i < sample.data.length; i++) { //Insert 700 random datasets into table bookings. (REFER TO SCHEMA IF NEEDED).
```
Comment (VincentCastelli) [inline database/db.js]: Please remove comment.

File: database/db.js [hunk 3]
```diff
@@ -3,26 +3,25 @@ var sample = require('./exampleData.js'); //sample data.
...
-	connection.query(q, [sample.data[i].partySize, sample.data[i].date, time[randomRest(0,46)], randomRest(1,100)], function(err, results) {
+	connection.query(q, [sample.data[i].partySize, sample.data[i].date, time[randomRest(0,35)], randomRest(1,100)], function(err, results) {
 		if (err) {
 			console.log("ERR in db.js database", err);
```
Comment (VincentCastelli) [inline database/db.js]: No console logs in error handling. Please handle this in a better way.

**Review by VincentCastelli — CHANGES_REQUESTED** [review]
Please fix the comments provided. Overall great job buddy!

**Review by Busyreadingsomething — COMMENTED** [review]
So your node modules were merged into the file. You need to get rid of those. For some reason, they got committed even though you ignored them in your .gitIgnore

**Pull request closed without merging**